include/pmacc/simulationControl/SimulationHelper.hpp

```diff
@@ -379,12 +406,19 @@ namespace pmacc
          *                 initial step to runSteps */
         uint32_t softRestarts;
 
-        /* period for checkpoint creation */
+        /* period for checkpoint creation [interval(s) based on steps]*/
         std::string checkpointPeriod;
 
         /* checkpoint intervals */
         SeqOfTimeSlices seqCheckpointPeriod;
 
+        /* period for checkpoint creation [period in seconds]*/
+        std::uint64_t checkpointTimePeriod;
```

Comment:
Seconds or minutes?

Also, perhaps this unit may become part of variable name here, and `checkpointPeriod` may become `checkpointPeriodSteps`

Reply:
done